apps/prairielearn/src/lib/copy-content.ts

```diff
@@ -163,7 +163,6 @@ async function initiateFileTransfer({
     storage_filename: path.join(relDir, f.slice(6)),
   };
 
-  if (config.filesRoot == null) throw new Error('config.filesRoot is null');
```

Comment:
A parsed config ensures this is always a string. Similarly for other config guards.